### ISSUE TITLE
Thoughts on adding $schema property to empty-fresh.json

### ISSUE THREAD

**Issue by jarz** [hacksalot/HackMyResume#79]
I found that adding this to my json made it easier to edit in VS Code.

"$schema": "https://raw.githubusercontent.com/fluentdesk/FRESCA/master/schema/fresh-resume-schema.json"

**Comment by hacksalot** [comment]
Agreed. Feel free to submit a PR or I'll roll this into the next release. Thanks! (and happy New Year)

**Comment by jarz** [comment]
I'm not sure what URL should be used. I don't know if editors will cache the schema and not catch updates.

**Comment by hacksalot** [comment]
The schema is GitHub-backed but the official/canonical location is currently:

http://fluentdesk.com/fresh/0.2.4/schema.json
http://fluentdesk.com/fresh/latest/schema.json

This will probably move to `.org` or dedicated domain in the future, but for now, that's the address of record. Probably best to use the specific `/0.2.4/` version as a cache-buster, especially since we control `empty-fresh.json` and can keep it synced.
